### DISCUSSION TITLE
How to use this?

### DISCUSSION THREAD

**Discussion by Anixx** [conradOU/readOutLoudSubtitlesFirefoxExtension#10] [Q&A]
I have installed it in Firefoox, choose /Microsoft voice, enabled subtitles in Youtube, but do not hear anything. Win11.

**Comment by conradRz** [comment]
Thanks for feedback.

remember to tick 
![image](https://github.com/conradRz/readAloudSubtitlesFirefoxExtension/assets/26015206/f8c61a21-acbf-4157-9f93-cbc80ae0179f)
checkbox underneath the videos. This is on purpose, as you wouldn't want it enabled for all the videos. 

I tested everything on a new Firefox profile just now, and everything works on mine. 

I can only presume, that by "enabled subtitles in Youtube" you had meant clicking 
![image](https://github.com/conradRz/readAloudSubtitlesFirefoxExtension/assets/26015206/2fcc65d3-fb47-45da-b990-6ea7d40a0551)
which is unnecessary.

I'll probably end up doing some sort of pop-up (on first use) instructions to remind folk to tick that checkbox, that's totally doable (it's very easy to detect first addon run). Although I "work" on priority basis, and have some other interesting ideas to develop/uni coursework/full-time job/job applications which have a higher priority.
